LICENSE

```diff
@@ -187,7 +192,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2020 cPanel, L.L.C.
+   Copyright [yyyy] [name of copyright owner]
```

Comment:
^^ Should this template be updated?

Reply:
https://metacpan.org/source/LEONT/Software-License-0.103014/lib%2FSoftware%2FLicense%2FApache_2_0.pm#L231

Reply:
issue reported as https://github.com/Perl-Toolchain-Gang/Software-License/issues/76